models/BaseProxy.cfc

```diff
@@ -213,4 +213,43 @@ component accessors="true" {
 		return ( findNoCase( "ForkJoinPool", getThreadName() ) NEQ 0 );
 	}
 
+	void function sendExceptionToLogBoxIfAvailable( required any exception ){
+		if ( !variables.loadAppContext ) {
+			return;
+		}
+
+		if ( !structKeyExists( application, "cbController" ) ) {
+			return;
+		}
+
+		try {
+			application.cbController
+				.getLogBox()
+				.getRootLogger()
+				.error( arguments.exception.message, arguments.exception );
```

Comment:
I see that the `Injector` has a LogBox instance.  So how do you feel about:
```suggestion
		if ( !structKeyExists( application, "wirebox" ) ) {
			return;
		}

		try {
			application.wirebox
				.getLogBox()
				.getRootLogger()
				.error( arguments.exception.message, arguments.exception );
```

Reply:
cc @lmajano

Reply:
Yes, that will work best.